### ISSUE TITLE
Selected tab is misleading

### ISSUE THREAD

**Issue by amey-kudari** [virtual-labs/computer-programming-responsive-iiith#245]
[BUG] [User-experience] 
![Screenshot from 2019-03-16 20-11-57](https://user-images.githubusercontent.com/43315144/54476864-dff7e380-4827-11e9-8c07-b778cb443c90.png)
currently, theory tab is open, but introduction is allways blue, so this can confuse user, making him think its still on introduction.

Suggested fix:
make selected tab in a different color.

**Comment by AnimeshSinha1309** [comment]
Is anyone working on this issue, or is it okay for me to take it? Amey, doing this?

**Comment by amey-kudari** [comment]
Yep @AnimeshSinha1309. I have a branch ready. I was but I we can work on it together if you wait. PR on my branch of you want to modify. (haven't yet pushed I'll let you know when i do)

**Comment by AnimeshSinha1309** [comment]
That's all right, I will work on other issues, thanks a lot.
